### PR TITLE
WC: Ecommerce Cards

### PR DESCRIPTION
**Pull request by Prabhav-Sharma**
Added Cards for eCommerce project.

### CODE REVIEW

File: Documentation/Components/E-commerce Card/commerce-card.css [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+    align-items: center;
+    font-family: 'Open Sans', sans-serif;
+    gap:var(--space-xsmall);
+}
+
+.prod-title{
+font-size: 1.25rem;
+line-height: 1.5rem;
+margin:1px;
+}
```
Comment (amarphule) [inline Documentation/Components/E-commerce Card/commerce-card.css]: you can use indentation

File: Documentation/Components/E-commerce Card/commerce-card.css [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+    gap:var(--space-small);
+    position: relative;
+}
+
+.badge-container{
+    width: 1.5rem;
+    height: 1.75rem;
+    position: absolute;
+    background-color: white;
+    border-radius: 999px;
```
Comment (nerdyspook) [inline Documentation/Components/E-commerce Card/commerce-card.css]: just a suggestion, you can put border-radius: 50% or border-radius: 100% if you want it round.

**Review by nerdyspook — CHANGES_REQUESTED** [review]

File: Documentation/Components/E-commerce Card/commerce-card.html [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+            <div class="right-section flex-column">
+            <div class="text-container flex-column">
+                <p class="prod-title">Men's Premium Jacket</p>
+                <div class="prod-costs flex-row">
+                <p class="prod-cost">₹1999</p>
+                <p class="prod-mrp">₹3999</p>
+                </div>
+                <p class="prod-discount">50%</p>
+            </div>
+            <div class="btns-95 flex-column">
```
Comment (amarphule) [inline Documentation/Components/E-commerce Card/commerce-card.html]: doesn't understand `btns-95 `meaning

File: Documentation/Components/E-commerce Card/commerce-card.css (hunk 2 above)
Comment (Prabhav-Sharma) [reply Documentation/Components/E-commerce Card/commerce-card.css]: Made the change, thank you

File: Documentation/Components/E-commerce Card/commerce-card.css [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+    padding:3px;
+    gap:var(--space-small);
+    position: relative;
+}
+
+.badge-container{
+    width: 1.5rem;
+    height: 1.75rem;
+    position: absolute;
+    background-color: white;
```
Comment (amarphule) [inline Documentation/Components/E-commerce Card/commerce-card.css]: you can make a variable for ` white` color and use it to keep consistency

File: Documentation/Components/E-commerce Card/commerce-card.css [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+    outline:1px solid black;
+}
+
+.overlay-text{
+    position: absolute;
+    z-index: 1;
+    top:30%;
+    padding: 2px 40%;
+    width:fit-content;
+    background-color: white;
```
Comment (amarphule) [inline Documentation/Components/E-commerce Card/commerce-card.css]: same as above

**Review by amarphule — CHANGES_REQUESTED** [review]

File: Documentation/Components/E-commerce Card/commerce-card.html (hunk 3 above)
Comment (Prabhav-Sharma) [reply Documentation/Components/E-commerce Card/commerce-card.html]: It was for the width of the button, I have added width to the class name now.

File: Documentation/Components/E-commerce Card/commerce-card.css (hunk 4 above)
Comment (Prabhav-Sharma) [reply Documentation/Components/E-commerce Card/commerce-card.css]: Yes, changed it to variable now. Thanks

File: Documentation/Components/E-commerce Card/commerce-card.css (hunk 1 above)
Comment (Prabhav-Sharma) [reply Documentation/Components/E-commerce Card/commerce-card.css]: indented the code

**Pull request merged**